src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      action="#"
+    >
+      <input
+        class="search_input_small form"
```

Comment:
Similarly, 'search_input_small' should be renamed to a class name that represents the meaning of the content.